src/view_text.cpp

```diff
@@ -356,6 +356,10 @@ void View::DrawText(DeviceContext *dc, Text *text, TextDrawingParams &params)
 
     dc->StartTextGraphic(text, "", text->GetUuid());
 
+    if(dc->GetFont()->GetWeight() != FONTWEIGHT_NONE && dc->GetFont()->GetStyle() != FONTSTYLE_NONE) {
```

Comment:
Please use `HasWeight()` and `HasStyle()` instead

Reply:
`FontInfo `doesn't have such methods. Should I implement them?

Reply:
Ah, sorry, I thought it was checking against an attribute class. You can leave it like this